functions-and-operators/string-functions.md

```diff
@@ -1229,19 +1229,38 @@ SELECT LPAD('TiDB',-2,'>');
 
 ### [`OCTET_LENGTH()`](https://dev.mysql.com/doc/refman/8.0/en/string-functions.html#function_octet-length)
 
-与 `LENGTH()` 功能相同
+与 [`LENGTH()`](#length) 功能相同
 
 ### [`ORD()`](https://dev.mysql.com/doc/refman/8.0/en/string-functions.html#function_ord)
 
 返回该参数最左侧字符的字符编码
 
 ### [`POSITION()`](https://dev.mysql.com/doc/refman/8.0/en/string-functions.html#function_position)
 
-与 `LOCATE()` 功能相同
+与 [`LOCATE()`](#locate) 功能相同
 
 ### [`QUOTE()`](https://dev.mysql.com/doc/refman/8.0/en/string-functions.html#function_quote)
 
-使参数逃逸，为了在 SQL 语句中使用
+`QUOTE()` 函数用于转义字符串，使其可以在 SQL 语句中使用。
+
+如果输入参数为 `NULL`，该函数返回 `NULL`。
+
+示例：
+
```

Comment:
英文版 PR https://github.com/pingcap/docs/pull/16958/files 在这里增加了一行内容，请看下是否保持一致：

To display the result directly instead of showing a hexadecimal-encoded value, you need to start the MySQL client with the [`--skip-binary-as-hex`](https://dev.mysql.com/doc/refman/8.0/en/mysql-command-options.html#option_mysql_binary-as-hex) option.